cookbook/jackrabbit.rst

```diff
@@ -0,0 +1,74 @@
+Using Jackrabbit
+================
+
+If you maintain a bigger website, it might make sense to use Jackrabbit instead of
+the ``doctrine-dbal`` implementation of Jackalope. Jackrabbit performs better in many
+cases, and as a bonus it also supports :doc:`../bundles/page/versioning` of content.
+
+Installation
+------------
+
+Use the following command to install the ``jackrabbit`` adapter:
+
+.. code-block:: bash
+
+   composer require jackalope/jackalope-jackrabbit
+
+In addition to the previous command you also have to make sure that Jackrabbit is running
+on your server.
+
+Configuration
+-------------
+
+Change the ``config/packages/sulu_document_manager.yaml`` file to something similar as
+below. Mind that it is recommended to pass the URL via an environment variable, which
```

Comment:
I would add a link to the Symfony documentation explaining their environment variables.